serverlessworkflow/modules/ROOT/pages/eventing/consume-producing-events-with-kafka.adoc

```diff
@@ -33,53 +26,59 @@ If using Maven, this means including following snippet in pom.xml:
+[[con-sw-smallrye-channel-configuration]]
+== Smallrye channels configuration for a Serverless Workflow
 
-Channels are defined by using link:{quarkus_config_url}[Quarkus configuration]. The pattern for channel properties is `mp.messaging.[incoming|outgoing].<channel name>.<property_name>`.
+You can configure Smallrye channels for a Severless Workflow using event definitions. The Smallrye channels are defined using link:{quarkus_config_url}[Quarkus configuration]. The format for Smallrye channel properties is `mp.messaging.[incoming|outgoing].<channel name>.<property_name>`. 
 
-Kogito serverless workflow allows different channel mapping strategies:
+Serverless Workflow allows the following channel mapping strategies:
 
 * Define one default incoming channel to receive all the incoming messages and one default outgoing channel to store all the published messages.
+* Define a channel for each link:{cloud_events_url}[CloudEvent] type so that every message type has a dedicated channel.
+* Define a channel for certain CloudEvent types. Allow the non-mapped CloudEvent types to use the default incoming or outgoing channel.
 
-* Define a channel for each link:{cloud_events_url}[Cloud Event] Type so that every message type has a dedicated channel.
+Kogito first searches for a channel name in the properties that is same as CloudEvent type. If the channel name is found, Kogito uses the channel for that CloudEvent type. However, if the channel name is not found, Kogito searches for the default channel definition. In case the default channel definition is not existing, then an error is reported. 
 
-* Define a channel for certain Cloud Event Types and let the not mapped Cloud Event types to use the default incoming or outgoing channel.
+The default incoming channel is `kogito_incoming_stream` and the default outgoing channel is `kogito_outgoing_stream`.
 
-Kogito first searches for channel name equals to Cloud Event type in the properties. If found, it uses that channel for that Cloud Event type. If not, it searches for default channel definition. If also not existing, then an error will be reported.
-
-The name for the default incoming channel is `kogito_incoming_stream` and for the default outgoing channel is `kogito_outgoing_stream`.
-
-You can change the default incoming and outgoing topic name as follows:
+To change the default incoming and outgoing topic names, you can use the following properties:
 
+.Properties to change default incoming and outgoing topic names
 [source,properties]
 ----
 kogito.addon.messaging.incoming.defaultName=<default channel name>
 kogito.addon.messaging.outgoing.defaultName=<default channel name>
 ----
 
-This is the full list of link:{kafka_connector_properties_url}[properties] you can use for a channel when using Kafka connector.
+The following properties are mandatory that you need to set for each channel:
```

Comment:
Shoud not it read
"The following properties are mandatory for each channel". "mandatory" and "need to set" sound redundant to me

Reply:
Done